virocon/plotting.py

```diff
@@ -190,32 +193,75 @@ def plot_dependence_functions(model, semantics=None, par_rename={}, axes=None):
     for dim in conditional_dist_idc:
         dist = model.distributions[dim]
         conditioning_values = dist.conditioning_values
-        x = np.linspace(0, max(conditioning_values))
+        if conditioning_values is not None:
+            x = np.linspace(0, max(conditioning_values))
+        else:
+            x = np.linspace(0, 10)
         cond_idx = model.conditional_on[dim]
         x_name = semantics["names"][cond_idx]
         x_symbol = semantics["symbols"][cond_idx]
         x_unit = semantics["units"][cond_idx]
         x_label = f"{x_name}," + r" $\it{" + f"{x_symbol}" + r"}$" + f" ({x_unit})"
         for par_name, dep_func in dist.conditional_parameters.items():
-            par_values = [par[par_name] for par in dist.parameters_per_interval]
             ax = axes[axes_counter]
             axes_counter += 1
-            ax.scatter(conditioning_values, par_values, c="k", marker="x")
-            ax.plot(x, dep_func(x), c="#004488")
+            # If a model is created directly (without fitting it to a dataset), it does
+            # not have conditioning values.
+            if conditioning_values is not None:
+                par_values = [par[par_name] for par in dist.parameters_per_interval]
+                ax.scatter(
+                    conditioning_values,
+                    par_values,
+                    c="k",
+                    marker="x",
+                    label="estimates from intervals",
+                )
+            if dep_func.latex is not None:
+                dep_func_label = dep_func.latex
+
+                # Replace 'x' with variable symbol (e.g. 'h_s')
+                splitted_symbol = x_symbol.split("_")
+                if len(splitted_symbol) == 1:  # If there was no underscore.
+                    var_symbol = splitted_symbol[0].lower()
+                else:  # If there was one or many underscores.
+                    var_symbol = (
+                        splitted_symbol[0].lower() + "_" + "_".join(splitted_symbol[1:])
+                    )
+
+                # Replace 'x' if it is not part of '\exp' which is checked by checking whether
+                # 'x' follows '\e'.
+                dep_func_label = re.sub(
+                    r"(?<!\\e)x", "{" + var_symbol + "}", dep_func_label
+                )
+
+                # Replace symbols of parameters (a, b, ..) with estimated values.
+                for par_name_local, par_value_local in dep_func.parameters.items():
+                    dep_func_label = dep_func_label.replace(
```

Comment:
Hmm... I considered this solution too, but thought "nah, at the moment we don't have such a dependence function and if the user has such a problematic dependence function he or she can still add curly brackets in the latex string to solve the problem". I still think so. In case one runs into problems, one can add the curly brackets to solve it, but we don't need them for all other dependence functions where this is not a problem.

Reply:
But if we do not check if there are curly braces before replacing, it does not make a difference.
If the user writes 
```latex
$\fr{a}c{a}{2}$
```
right now both `a` are still replaced.

Reply:
What we could do is:
- search for one or multiple `{a}` in the string and replace accordingly
- if none is found search for `a` instead and replace the first occurence

Reply:
Great idea. As discussed on the phone I will open an issue for this such that we can implement it later and merge this PR now.